Bugzilla/Template.pm

```diff
@@ -401,7 +401,13 @@ sub css_files {
 
     unshift @requested_css, "skins/yui.css" unless $no_yui;
 
-    my @css_sets = map { _css_link_set($_) } @requested_css;
+    my @css_sets = map { _css_link_set($_) } sort {
```

Comment:
I don't mind that the sort may be inefficient, but I'd love to see a comment explaining why this sort is required.